engine.go

```diff
@@ -26,6 +29,34 @@ type Engine struct {
 	Auth     auth.Auth
 }
 
+var (
+	// QueryCounter describes a metric that accumulates number of queries monotonically.
+	QueryCounter = discard.NewCounter()
+
+	// QueryErrorCounter describes a metric that accumulates number of failed queries monotonically.
+	QueryErrorCounter = discard.NewCounter()
+
+	// QueryHistogram describes a queries latency.
+	QueryHistogram = discard.NewHistogram()
+)
+
+func observeQuery(ctx *sql.Context, query string) func(err error) {
+	logrus.WithField("query", query).Debug("executing query")
+	span, _ := ctx.Span("query", opentracing.Tag{Key: "query", Value: query})
```

Comment:
It's safe to ignore the possible error here?

Reply:
it's not an error, the second returned value is context